tedana/metrics/collect.py

```diff
@@ -151,13 +150,13 @@ def generate_metrics(
 
     if ("map FT2" in required_metrics) or ("map FS0" in required_metrics):
         LGR.info("Calculating F-statistic maps")
-        m_T2, m_S0, p_m_T2, p_m_S0 = dependence.calculate_f_maps(
+        m_t2, m_s0, p_m_t2, p_m_s0 = dependence.calculate_f_maps(
```

Comment:
The style guide clearly doesn't like underscore than a capital letter, but, in this case, T2 and S0 are capitalized acronyms. If possible, it would be good to make an exception to keep them capitalized, but, if creating such an exception is non-trivial, I'm fine approving this change.

(Same issue in lines 222-224, 278-280, & 326-352)